scripts/jah/step.lua

```diff
@@ -210,6 +244,14 @@ function init()
   timer:start()
 end
 
+function cleanup()
+  if grid_device then
+    grid_device:all(0)
+    grid_device:refresh()
```

Comment:
i think this can be removed since grid cleanup happens in `script.lua` already?

https://github.com/monome/norns/blob/11e38b8b6562a65bd347692086aa77ca699ab8a1/lua/script.lua#L18-L21